src/app/components/search/Search.tsx

```diff
@@ -17,14 +17,14 @@ export default function Search({
 
   return (
     <>
-      {displayCustomCourseSelection && <CustomCourseSelection />}
       <Card
```

Comment:
Search card should have a heading and possibly a `help` icon to hover over with some helpful tips on how to use it